colossalai/nn/optimizer/cpu_adam.py

```diff
@@ -0,0 +1,109 @@
+# modified from https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/ops/adam/cpu_adam.py
+
+import math
+import torch
+import time
+from pathlib import Path
+import colossalai
+
+
+class CPUAdam(torch.optim.Optimizer):
+    optimizer_id = 0
+
+    def __init__(self,
+                 model_params,
+                 lr=1e-3,
+                 bias_correction=True,
+                 betas=(0.9,
+                        0.999),
+                 eps=1e-8,
+                 weight_decay=0,
+                 amsgrad=False,
```

Comment:
Hi, I find `amsgrad` here, but you don't initialize a `max_exp_avg_sq` buffer. Do we support `amsgrad` mode now?

Reply:
Thanks to point out it. We don't support the mode. I would delete these segments.